WordPress/Classes/ViewRelated/Reader/ReaderRetryFailedImageView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ReaderRetryFailedImageView.swift
+//  WordPress
+//
+//  Created by Paul Von Schrottky on 10/5/19.
+//  Copyright © 2019 WordPress. All rights reserved.
+//
+
+import UIKit
+import Gridicons
+
+protocol ReaderRetryFailedImageDelegate: AnyObject {
+    func didTapRetry()
+}
+
+fileprivate class LinkTextView: UITextView {
+    override func selectionRects(for range: UITextRange) -> [UITextSelectionRect] {
+        return []
+    }
+
+    override func caretRect(for position: UITextPosition) -> CGRect {
+        return CGRect.zero.offsetBy(dx: .greatestFiniteMagnitude, dy: .greatestFiniteMagnitude)
+    }
+}
+
+class ReaderRetryFailedImageView: UIView {
```

Comment:
I agree this is lacking explanation. Fortunately, I'll be removing this class because we're no longer going to need the link itself to be tappable now that the retry view itself will be the touch target.
For completeness' sake though - this was to disable selection on the text view. Allowing tappable links necessitated `isSelectable` to be `true`, which has the undesirable result here of allowing text selection. Casting the cursor caret into the far reaches of the coordinate space didn't appeal to me, but was the most reliable hack after lots of testing.

Reply:
I believe you're talking about `LinkTextView`. I meant for `ReaderRetryFailedImageView`. 🙂

Reply:
Thanks! I added a short but descriptive comment.